pkgs/top-level/stage.nix

```diff
@@ -87,6 +94,8 @@ let
   stdenvBootstappingAndPlatforms = self: super: {
     buildPackages = (if buildPackages == null then self else buildPackages)
       // { recurseForDerivations = false; };
+    __targetPackages = (if __targetPackages == null then self else __targetPackages)
```

Comment:
Err it would normally be an attribute set. I wanted to distinguish `{}` --- there are no target packages in the set, from `null` -- there is no target package set whatsoever.